packages/cli-exec/src/exec.js

```diff
@@ -120,7 +116,9 @@ async function* spawn(cmd, args, percy) {
 
     if (proc.stderr) {
       proc.stderr.on('data', (data) => {
-        errorMessage += data.toString();
+        const message = data.toString();
+        let entry = { message, timestamp: Date.now(), type: 'ci' };
+        percy.log.error(entry, null, true);
```

Comment:
when percy is disabled then it will throw an error.
Add test case for this case as well.